rpcs3/Emu/Cell/lv2/sys_spu.h

```diff
@@ -233,9 +233,10 @@ struct lv2_spu_group
 	atomic_t<u32> run_state; // SPU Thread Group State
 	atomic_t<s32> exit_status; // SPU Thread Group Exit Status
 	atomic_t<u32> join_state; // flags used to detect exit cause
+	volatile bool join_signal; // flag used to signal group_join
 	atomic_t<u32> running; // Number of running threads
-	cond_variable cond; // used to signal waiting PPU thread
-	atomic_t<u64> stop_count;
+	cond_variable cond; // used to signal waiting PPU thread 
+	volatile u64 stop_count;
```

Comment:
Don't use volatile anywhere. Volatile != atomic, it has no such guarantees and is unnecessary for us in general.